packages/admin-ui/src/number-input/number-input.tsx

```diff
@@ -66,8 +66,12 @@ export const NumberInput = forwardRef(
     }
 
     return (
-      <FormControl error={error} optional={optional}>
-        {label && <FormControlLabel htmlFor={id}>{label}</FormControlLabel>}
+      <FormControl>
+        {label && (
+          <FormControlLabel optional={optional} htmlFor={id}>
+            {label}
+          </FormControlLabel>
+        )}
```

Comment:
TBH I don't see the need for a ternary here since `label` is not an array but I've seen that the `select` component follows this pattern (just to keep things standardized) 

```suggestion
        {label ? (
          <FormControlLabel optional={optional} htmlFor={id}>
            {label}
          </FormControlLabel>
        ) : null}
```

Reply:
Well, text-area and text-input also use && 😄

Reply:
Yes!